src/cpu/CPUCore.cc

```diff
@@ -518,14 +519,32 @@ static constexpr void toHex(byte x, std::span<char, 3> buf)
 	buf[1] = toHex(x & 15);
 }
 
+template<typename T> void CPUCore<T>::disasmBlobCommand(
+	Interpreter& interp, std::span<const TclObject> tokens, TclObject& result) const
+{
+	if (tokens.size() < 4) {
+		interp.wrongNumArgs(2, tokens, "value addr");
+	}
+	std::string dasmOutput;
```

Comment:
Very very much a detail:
In general prefer to declare local variables as close as possible to where they are first needed.
In this case you construct an empty string `dasmOutput`. Then you check for (more) possible errors. So if that error triggers constructing the string was for nothing.
This is easily avoided by moving this line right before the call to `dasm()`.

So very much a detail: it can be a small performance cost (in corner cases). But more important to also improves the readability of the code.